lib/geometry/polygon.nit

```diff
@@ -19,9 +19,8 @@ module polygon
 
 import points_and_lines
 
-# Convex Polygon class
-class ConvexPolygon
-
+# Abstraction of a Polygon
+abstract class APolygon
```

Comment:
I do not remember seeing the `A` used as prefix of abstract classes in Nit. I do not mind in fact, it is just an information.